src/auro-nav.js

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) 2022 Alaska Airlines. All right reserved. Licensed under the Apache-2.0 license
+// See LICENSE in the project root for license information.
+
+// ---------------------------------------------------------------------
+
+/* eslint-disable max-classes-per-file, array-bracket-newline, max-params */
+
+// If using litElement base class
+import { LitElement, html } from "lit";
+
+// If using auroElement base class
+// See instructions for importing auroElement base class https://git.io/JULq4
+// import { html, css } from "lit-element";
+// import AuroElement from '@alaskaairux/webcorestylesheets/dist/auroElement/auroElement';
+
+// Import touch detection lib
+import "focus-visible/dist/focus-visible.min.js";
+import styleCss from "./style-css.js";
+import styleCssFixed from './style-fixed-css.js';
+import styleBreadcrumbCss from "./style-breadcrumb-css.js";
+import styleCssAuroHyperlink from "@aurodesignsystem/auro-hyperlink/src/style-css";
+
+import { AuroHyperlink } from "@aurodesignsystem/auro-hyperlink/src/auro-hyperlink.js";
+
+// See https://git.io/JJ6SJ for "How to document your components using JSDoc"
+/**
+ * The auro-nav element provides users a way to ... (it would be great if you fill this out).
+ *
+ * @attr {Boolean} homeIcon - Adds a 'home' auro-icon within the first auro-hyperlink when the attribute is present.
+ * @slot Slot for insertion of auro-hyperlinks.
+ */
+
+// build the component class
+export class AuroNav extends LitElement {
+  constructor() {
+    super();
+
+    /* eslint-disable no-unused-expressions */
+    customElements.get('auro-nav-breadcrumb') || customElements.define('auro-nav-breadcrumb', class extends AuroHyperlink {
```

Comment:
```suggestion
    customElements.get('auro-breadcrumb') || customElements.define('auro-breadcrumb', class extends AuroHyperlink {
```

The name `auro-nav-breadcrumb` does not compliant with [Auro's naming convention](https://github.com/AlaskaAirlines/WC-Generator/blob/master/bin/generate.js#L73) guidelines.

Reply:
This is an interesting new concept. I want to discuss this as a group to ensure we are not simply using a shiny new tool and that there is a plan for expansion. E.g. what if there is a new 'nav' element? How is that going to be supported? 

What is the real value of `auro-nav`? To create `auro-breadcrumb` on the fly? Then `auro-breadcrumb` cannot be used outside of `auro-nav`?